src/pybind/mgr/dashboard_v2/tests/test_tcmu_iscsi.py

```diff
@@ -0,0 +1,107 @@
+from cherrypy.test.helper import CPWebCase
+import cherrypy
+import mock
+
+from ..controllers.auth import Auth
+from ..tools import SessionExpireAtBrowserCloseTool
+from ..controllers.tcmu_iscsi import TcmuIscsi
+from .helper import ControllerTestCase
+
+mocked_servers = [{
+    'ceph_version': 'ceph version 13.0.0-5083- () mimic (dev)',
+    'hostname': 'ceph-dev',
+    'services': [{'id': 'a:b', 'type': 'tcmu-runner'}]
+}]
+
+mocked_metadata = {
+    'ceph_version': 'ceph version 13.0.0-5083- () mimic (dev)',
+    'pool_name': 'pool1',
+    'image_name': 'image1',
+    'image_id': '42',
+    'optimized_since': 100.0,
+}
+
+mocked_get_daemon_status = {
+    'lock_owner': 'true',
+}
+
+mocked_get_counter = {
+    'librbd-42-pool1-image1.lock_acquired_time': [[10000.0, 10000.0]],
+    'librbd-42-pool1-image1.rd': 43,
+    'librbd-42-pool1-image1.wr': 44,
+    'librbd-42-pool1-image1.rd_bytes': 45,
+    'librbd-42-pool1-image1.wr_bytes': 46,
+}
+
+mocked_get_rate = {
```

Comment:
Apparently this mock is incorrect.

Looking into ["get_rate" code](https://github.com/ceph/ceph/blob/master/src/pybind/mgr/dashboard/module.py#L247), I'd say that this method should return a number, and not a dictionaty. Am I wrong?